install.sh

```diff
@@ -6,11 +6,11 @@ set -o nounset
 set -o xtrace
 
 if [[ "${LANGUAGE}" == "cwl" ]]; then
-    curl -o requirements.txt "https://raw.githubusercontent.com/dockstore/dockstore/1.7.0-rc.3/dockstore-webservice/src/main/resources/requirements/1.7.0/requirements3.txt"
+    curl -o requirements.txt "https://raw.githubusercontent.com/dockstore/dockstore/1.13.0-beta.3/dockstore-webservice/src/main/resources/requirements/1.13.0/requirements3.txt"
     pip3 install --user -r requirements.txt
 elif [[ "${LANGUAGE}" == "wdl" ]]; then
     wget https://github.com/broadinstitute/cromwell/releases/download/44/cromwell-44.jar
```

Comment:
Do we need this? We don't have instructions on how to run directly with Cromwell, and the CLI is going to download its own version of Cromwell, not use this one.

Reply:
Actually, I only looked at the diff, so perhaps we do reference it from unmodified code. Maybe update to the same version the CLI uses if so though.

Reply:
I think I'd probably clean this up in `can also add dockstore cli testing, but I think this is getting out of scope, so will create a ticket on merge`

Reply:
(i.e. we can test with cromwell for now just to make sure it is valid WDL and works, then when I switch to dockstore CLI, it will automatically be running the "right" version)